packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -334,6 +334,149 @@ void main() {
     expect(tester.getBottomLeft(find.text('Title')).dy, 44.0 - 8.0); // Extension gone, (static part - padding) left.
   });
 
+  testWidgets(
```

Comment:
can we move this below 'Border should not be displayed when null' to keep things in order?